src/main/java/io/github/darealturtywurty/turtychemistry/core/multiblocks/MultiblockListener.java

```diff
@@ -17,44 +17,54 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 import net.minecraftforge.fml.common.Mod;
 import org.apache.commons.lang3.tuple.Pair;
+import org.apache.logging.log4j.Level;
 import org.jetbrains.annotations.Nullable;
 
 @Mod.EventBusSubscriber(modid = TurtyChemistry.MODID, bus = Mod.EventBusSubscriber.Bus.FORGE)
 public final class MultiblockListener {
     @SubscribeEvent
     public static void blockPlace(BlockEvent.EntityPlaceEvent event) {
+        //cache the event caller's Level and Block Position
+
         if(!(event.getEntity() instanceof Player) || event.getLevel().isClientSide())
             return;
-
+        final LevelAccessor currentEventLevel = event.getLevel();
+        final BlockPos currentEventPosition = event.getPos();
+        final BlockState currentEventPlacedBlock = event.getPlacedBlock();
```

Comment:
```suggestion
        final LevelAccessor level = event.getLevel();
        final BlockPos position = event.getPos();
        final BlockState block = event.getPlacedBlock();
```
These names were really unnecessarily descriptive. You will also have to change the below usages of this too ofc.